### ISSUE TITLE
Game Profile won't recognise Skyrim special edition folder

### ISSUE THREAD

**Issue by Farshnuke** [matortheeternal/zedit#251]
I use MO2 and Wyre Bash SSEedit and even the old Merge Plugins can identify my skyrim special edition folder and mods just fine.
Zedit can identify Fallout 4 and its mods on the same drive so I don't know why this isn't working.

I can only think that the downgrade patcher is somehow interfering but I'm not going to test that since I need the patcher
![14112021](https://user-images.githubusercontent.com/10143083/141678888-9582d163-8577-49aa-8e36-72af545ba736.PNG)

**Comment by lieteau0522** [comment]
solved?

**Comment by Farshnuke** [comment]
Okay this problem recurred with Fallout 4. Or rather I was an idiot each time. When I first installed MO2 I moved Steam over to a different hard drive (one with more space and because the C drive is known to cause weirdness) Somehow I ended up with separate and duplicate Skyrim Special edition and Fallout 4 folders. One installed straight under steamapps. One installed under Steamapps and then Common

Clearly when it came time properly install mods in Skyrim Special edition I decided to use the newly created folder under common and then forgot as I had been used to modding Fallout 4 where my active directory is installed straight under steam apps.

This time around I I was used to working with Skyrim Special edition so could not fathom why none of the executables pointing at the Fallout 4 data folder installed under common were not recognising the mods when MO2 clearly was. The answer was that MO2 and all my mods are set up to use the Fallout 4 folder installed straight under steam apps.

TLDR The problem was caused by duplicate game installs and my idiot brain getting confused. 

This is why troubleshooting includes check everything is installed correctly. Unfortunately when there isn't actually a problem with either program the brain that knows how programs usually works goes "They must be installed correctly I followed the steps, everything is working" What extra messed with my head was the mods from the creative club by Bethesda downloaded to both versions of Fallout 4

So yes problem solved I am an idiot.
